easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -96,6 +96,9 @@ def configure_step(self, srcdir=None, builddir=None):
             if value is not None:
                 options.append("-D%s='%s'" % (option, value))
 
+        # show what CMake is doing by default
+        options.append("-DCMAKE_VERBOSE_MAKEFILE=ON")
```

Comment:
Why? The logging options already decide over that `run_cmd` should display. And I cannot think of a single reason why we wouldn't want the full output?